apps/ec.c

```diff
@@ -219,6 +219,11 @@ int ec_main(int argc, char **argv)
 
     if (no_public)
         EC_KEY_set_enc_flags(eckey, EC_PKEY_NO_PUBKEY);
```

Comment:
Our coding style says if one branch of the if has {} then they both do - so please could you add them here as well?